redirections/sessions/talks/django-database-gotchas.md

```diff
@@ -0,0 +1 @@
+new_url: handling-database-concurrency-with-django.md
```

Comment:
This doesn’t work for me if I run locally; I get redirected to http://localhost:8000/sessions/talks/django-database-gotchas/handling-database-concurrency-with-django.md